src/content/docs/infrastructure/install-infrastructure-agent/linux-installation/container-instrumentation-infrastructure-monitoring.mdx

```diff
@@ -1,19 +1,19 @@
 ---
 title: Container instrumentation for infrastructure monitoring
```

Comment:
@harrykimpel How do feel about making this title clearer by renaming it to "Monitor your containers with the infrastructure agent"?

Reply:
@akristen agreed, sounds great!